waku/waku_api/rest/doc/relayapi_auto_subscriptions.yaml

```diff
@@ -0,0 +1,50 @@
+# /relay/v1/auto/subscriptions:
+post: # post_waku_v2_relay_v1_auto_subscriptions
+  summary: Subscribe a node to an array of topics
+  description: Subscribe a node to an array of Content topics.
+  operationId: postSubscriptions
+  tags:
+    - relay
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type array:
+          items:
+            $ref: './schemas/apitypes.yaml#/ContentTopic'
```

Comment:
We don't seem to provide an option to specify minPeers required to publish a message. This is available in the relay API via programming interface. 
It would be good to specify this either in postMessage or subscribe as well.